config/detekt.yml

```diff
@@ -144,7 +144,7 @@ complexity:
     ignoreStringsRegex: '$^'
   TooManyFunctions:
     active: true
-    excludes: ['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**']
+    excludes: ['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**', '**/CompanyApi.kt', '**/CompanyDataController.kt']
```

Comment:
Should we document the addition of the two files somewhere so we don't forget to remove them again in the future?
Possible in the ticket concerning the restructurering of the APIs or in the manual maintanence template

Reply:
why are the files excluded from detekt?

Reply:
and yes, definately document it in MM

Reply:
Detekt flags them, because they have more than 10 functions/endpoints. In the daily today we brought it up and the team decided we should go ahead and ignore the finding for now, as we also have a task in the backend, in which we will reorganize and refactor our APIs. So we don't want to do it now and by this prevent unnecessary effort.